src/main/java/frc/robot/subsystems/drivetrain/Drivetrain.java

```diff
@@ -62,6 +63,7 @@ public Drivetrain() {
 
         kinematics = new DifferentialDriveKinematics(robotConstants.drivetrainConstants.kWheelBaseWidth);
         odometry = new DifferentialDriveOdometry(Rotation2d.fromDegrees(getAngle()));
+        SmartDashboard.putData("Drivetrain/DifferentialDrive", drivetrain);
```

Comment:
SmartDashboard.putData("Drivetrain/DifferentialDrive", drivetrain);
do as tuning method, and use it if the DriveWithXbox command gets true in constructor for tuning mode